client/packages/invoices/src/Prescriptions/DetailView/Payments/PaymentsModal.tsx

```diff
@@ -61,23 +67,69 @@ export const PaymentsModal: FC<PaymentsModalProps> = ({
           }}
         />
       }
+      sx={{
+        '& .MuiDialogContent-root': { display: 'flex', alignItems: 'center' },
+      }}
     >
-      <Grid container spacing={3} justifyContent="center">
-        {fields.map(({ label, value, disabled = false, onChange }, index) => (
-          <Grid key={index} size={4}>
-            <InputWithLabelRow
-              label={label}
-              Input={
-                <CurrencyInput
-                  value={value}
-                  disabled={disabled}
-                  onChangeNumber={onChange}
-                />
+      <Stack gap={2}>
+        <InputWithLabelRow
+          label={t('label.total-to-be-paid')}
+          Input={
+            <CurrencyInput
+              value={prescriptionData?.pricing.totalAfterTax}
+              onChangeNumber={() => {}}
+              style={{ borderRadius: 4, pointerEvents: 'none' }}
+            />
+          }
+        />
+        <InputWithLabelRow
+          label={t('label.provider-name')}
+          Input={
+            <Autocomplete
+              options={
+                insuranceData?.nodes.map(({ insuranceProviders }) => ({
```

Comment:
It's possible to have 2 active policies from the same provider. Can you add the policy number in here?
Maybe call it `Insurance Scheme`  as per mockup?

![image](https://github.com/user-attachments/assets/97a1dfb8-c7c0-44b0-9047-ec1196d499cf)